lib/src/ast/attribute.dart

```diff
@@ -2,13 +2,14 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:collection/collection.dart';
 import 'package:angular_ast/src/ast.dart';
 import 'package:angular_ast/src/token/tokens.dart';
 import 'package:angular_ast/src/visitor.dart';
 import 'package:source_span/source_span.dart';
 import 'package:quiver/core.dart';
 
-// TODO: Interpolation within the value
+final _listEquals = const ListEquality<dynamic>();
```

Comment:
I'd recommend making this a `ListEquality<Mustache>` and calling it like `_mustacheListEquals`